daemon/cmd/hubble.go

```diff
@@ -131,17 +124,22 @@ func (d *Daemon) launchHubble() {
 	}()
 
 	// configure another hubble instance that serve fewer gRPC services
-	if len(addresses) > 0 {
+	address := option.Config.HubbleListenAddress
+	if address != "" {
+		// TODO: remove warning once mutual TLS has been implemented
+		if !strings.HasPrefix(address, "unix://") {
```

Comment:
This `if` check can now be removed as a unix domain socket is no longer expected as an additional listen address (but leave the warning).

Reply:
fixed